adoc/retail_chap_install.adoc

```diff
@@ -238,21 +238,33 @@ Do not install the branch server as a traditionally managed client.
 
 === Assign and configure branch server formulas
 
-Before you configure the branch server formulas, ensure you have configured networking, and know the Salt ID of the branch server.
+Before you configure the branch server, ensure you have decided on networking topology, and know the Salt ID of the branch server.
 
 .Procedure: Configuring branch server formulas
 
 . Branch server configuration is performed using the [command]``retail_branch_init`` command:
 +
+.Configure branch server using default values and single shared network interface
```

Comment:
Needs a space between the dot and the step content.

Reply:
I do not mean it as a step content, but as a description of the example.
I want to show two different ways to call `retail_branch_init` script

Reply:
In that case, we probably need to review the procedure. Can you remove the example from within the procedure, and put it separately at the end? You don't want to mix up content types.